### ISSUE TITLE
Events single page multiple-dates dropdown need more color contrast

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#5495]
# Problem

https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html

Past dates on uiowa events single pages have insufficient color contrast.
~https://education.uiowa.edu/event/92671/0~
https://sandbox.prod.drupal.uiowa.edu/event/31316/21

Text color
 #999999 / rgb(153,153,153)
Background color
 #ffffff / rgb(255,255,255)
Contrast ratio
2.85 out of a required 4.5

WAVE shows a couple more examples of contrast ratio issues on elements that are not visible on the page. Do we need to address those as part of this, separate or do we not need to worry about those?

# Proposed solution

- Change to an accessible gray on white value.

**Comment by bspeare** [comment]
It might be worth doing https://github.com/uiowa/uiowa/issues/4916 with this work.

**Comment by briand44** [comment]
Grooming conversation... 

The contrast ratio issue seen in WAVE with the logo is outside of scope for this issue. 

Discussed deferring on #4916 until we have more conversations around that, especially with the new events system just launching. 

Sounds like this contrast ratio issue related to the past event dates can be resolved with a simple css change. We believe we have an accessible gray on white color already. Also, discussed possibly adding some additional spacing between the first date (date of event instance currently viewing) and the rest to reduce confusion around the order of these dates. Not required to resolve the accessibility issue, developer preference.

docroot/modules/custom/sitenow_events/css/single-event.css

Pick a replacement from docroot/themes/custom/uids_base/uids/scss/abstracts/_variables.scss
